uamqp/async_ops/connection_async.py

```diff
@@ -71,7 +70,7 @@ def __init__(self, hostname, sasl,
                  debug=False,
                  encoding='UTF-8',
                  loop=None):
-        self.loop = loop or get_running_loop()
+        self._internal_kwargs = get_dict_with_loop_if_needed(loop)
```

Comment:
same comment as before on the default get_running_loop() behavior for python 3.6 -> 3.9